python/lsst/analysis/drp/colorColorFitPlot.py

```diff
@@ -193,12 +206,7 @@ def colorColorFitPlot(self, catPlot, plotInfo, fitParams):
                        self.config.axisLabels["y"])))
 
         # Define a new colormap
-        r, g, b = colors.colorConverter.to_rgb("C0")
-        r1, g1, b1 = colors.colorConverter.to_rgb("midnightblue")
-        colorDict = {"blue": ((0.0, b, b), (1.0, b1, b1)), "red": ((0.0, r, r), (1.0, r1, r1)),
-                     "green": ((0.0, g, g), (1.0, g1, g1))}
-        colorDict["alpha"] = ((0.0, 0.0, 0.0), (0.05, 0.3, 0.3), (0.5, 0.8, 0.8), (1.0, 1.0, 1.0))
-        newBlues = colors.LinearSegmentedColormap("newBlues", colorDict)
+        newBlues = mkColormap(["paleturquoise", "midnightblue"])
 
         # Make a figure with three panels
         fig = plt.figure()
```

Comment:
As with other comments in this review, I think you need to set `dpi=300` here too.